streams/tests/test_core.py

```diff
@@ -270,6 +270,22 @@ def test_combine_latest():
 
     assert L == [(2, 'a'), (3, 'a'), (3, 'b')]
 
+def test_combine_latest_emit_on():
+    a = Stream()
+    b = Stream()
+    c = a.combine_latest(b, emit_on=0)
```

Comment:
I can imagine people putting `emit_on=a` or `emit_on=[a, b]` here as well.  This might be less ambiguous than numerical indexes.  I recommend accepting both forms of output and then normalizing down to numeric indexes if you have the interest/time.